tipCalculator/index.html

```diff
@@ -66,6 +68,14 @@ <h2>Tip calculation</h2>
         calculateTip(amt);
       });
 
+      button.addEventListener('mouseover', function(){
+      button.style.background = "rgb(255,127,80)";
```

Comment:
can we make a css class and toggle that? It will help in the future to update the style in the future without getting into JavaScript.

In HTML  <p id="amountText">fdfsdf</p> 
Replace the 'testing text' with some contextual text.

Reply:
Added separate css filse and updated all the changes to that files. Replaced the testing text too.
Yep, we can make different css class and toggle those with the help of the basics DOM commands.